### ISSUE TITLE
Leaderboard not sorted by date

### ISSUE THREAD

**Issue by ed-bristow** [vtp5/vtp5#140]
I don't know if this is the right place but look:
![image](https://cloud.githubusercontent.com/assets/11273941/15756593/d63cf1c8-28f9-11e6-894f-163760524841.png)

On the bottom right (in the previous score box) it says "14/05/16 17:46" is after "14/05/1618:30". So the scores are not in date order when it says they are.

**Comment by spudowiar** [comment]
Hmm, would be interesting to see the leaderboards database. I haven't dabbled in this area of the code however so I'll leave it to you guys to sort out :)

**Comment by leMingathon** [comment]
Well noticed! Yep - this is a known issue and will hopefully be resolved in the next release of VTP5 (which will definitely be available sometime before 1st January 2017 - right, guys? @nikunjpaliwal @Spagy @yousufahmed :)). I didn't work on the database, but I believe that the issue is that the dates and times are just stored as strings, so when they're "sorted", they're sorted into alphabetical/numerical order instead of chronological order.

Yeah, I know... We need to fix it.

**Comment by ghost** [comment]
See, this is why I believe we need more documentation and discussion. Changing a database schema is messy, but I'd much prefer to be using Unix timestamps.

**Comment by leMingathon** [comment]
@spudowiar I totally agree. Unfortunately, nobody is currently working on VTP5 (AS exams are ongoing, but we've also somewhat lost interest in the project...). I for one would really like to see VTP5 back in development, so hopefully once the exams are all over we'll start getting back into it...
